docs/services/slam/_index.md

```diff
@@ -31,3 +31,203 @@ Integrated SLAM Libraries include:
+Go to your robot's **Code sample** tab on the [Viam app](https://app.viam.com) for boilerplate code to connect to your robot.
+
+{{% /alert %}}
+
+### GetPosition
+
+Get the current position of the component the SLAM service is configured to source point cloud data from in the SLAM map as a [`Pose`](/internals/orientation-vector/).
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- [(spatialmath.Pose)](https://pkg.go.dev/go.viam.com/rdk@v0.3.0/spatialmath#Pose): A `Pose` representing the current position of the specified component.
+- [(string)](https://pkg.go.dev/builtin#string): The `"name"` of the component the SLAM service is configured to source point cloud data from.
+For example, a [camera](/components/camera/) named `"cam"`.
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/services/slam).
+
+```go {class="line-numbers linkable-line-numbers"}
+slam_svc, err := slam.FromRobot(robot, "my_slam_service")
+
+// Get the current position of the specified source component in the SLAM map as a Pose.
+pos, name, err := slam_svc.GetPosition(context.Background())
+```
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `timeout` [(Optional\[float\])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- [(Pose)](https://python.viam.dev/autoapi/viam/components/arm/index.html#viam.components.arm.Pose): A `Pose` representing the current position of the component the SLAM service is configured to source point cloud data from.
+For example, a [camera](/components/camera/) named `"cam"`.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/slam/client/index.html#viam.services.slam.client.SLAMClient.get_position).
+
+```python {class="line-numbers linkable-line-numbers"}
+slam_svc = SLAMClient.from_robot(robot=robot, name="my_slam_service")
+
+# Get the current position of the specified source component in the SLAM map as a Pose.
+pose = await slam.get_position()
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetPointCloudMap
+
+Get the point cloud map.
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+
+**Returns:**
+
+- (func() [[]byte](https://pkg.go.dev/builtin#byte), [error](https://pkg.go.dev/builtin#error)): The complete point cloud map in standard PCD format.
```

Comment:
@sguequierre is it right that `[error](https://pkg.go.dev/builtin#error))` should be on this line? I wasn't sure since the line description ("The complete point cloud map in standard PCD format") just refers to the `[]byte` part, and there's a line below about the error part

Reply:
I agree this is a bit confusing, I meant it to match the way the function return is typed here-- https://pkg.go.dev/go.viam.com/rdk/services/slam#Service -- do you think that's ok?

Reply:
Ohh I see, you're right! It's returning a function that takes no parameters and returns the byte array or an error. Resolving this and the other thread.